proxies/live/apiproxy/policies/AssignMessage.Errors.RequestTimeout.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage name="AssignMessage.Errors.RequestTimeout">
+    <Set>
+        <StatusCode>408</StatusCode>
+    </Set>
+    <AssignVariable>
+        <Name>pds.error.code</Name>
+        <Value>timeout</Value>
+    </AssignVariable>
+    <AssignVariable>
+        <Name>pds.error.coding.code</Name>
+        <Value>UNABLE_TO_CALL_SERVICE</Value>
+    </AssignVariable>
+    <AssignVariable>
+        <Name>pds.error.coding.display</Name>
+        <Value>Unable to call Service</Value>
+    </AssignVariable>
+    <AssignVariable>
+        <Name>pds.error.diagnostics</Name>
+        <Template>For a SyncAsync wrapped request, the downstream domain processing has not completed within the configured timeout period.</Template>
```

Comment:
This doesn't need to be `<template>`; can just be `<value>`

I can see the wording has come from https://nhsd-confluence.digital.nhs.uk/display/APM/2020-04-17+Spine+Error+Codes+II however it's not appropriate for public consumption - "SyncAsync wrapped..." is meaningless to our clients.

I'm sure I saw some wording, somewhere, about retrying the same `X-Request-ID`? I can't find it now though...but that's what the diagnostics/advice needs to be.

Reply:
I can't find the `X-Request-ID` referenced anywhere in confluence although I did have it in my previous message. Can I make an exec decision on this and fix forward?